server/models/user.js

```diff
@@ -23,12 +29,33 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: false,
       validate: { notEmpty: true }
     },
+    createdAt: {
+      type: DataTypes.DATE,
+      field: 'created_at'
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      field: 'updated_at'
+    },
     role: DataTypes.ENUM('ADMIN', 'TEAM_LEAD', 'DEVELOPER'),
-    office: DataTypes.STRING
+    office: DataTypes.STRING,
+    team: DataTypes.STRING
```

Comment:
I assume this is your way of denormalizing the database and having a simpler way of fetch the current team of the user. Why did you choose to make this a string field instead of having another relation with the team? And how much are you saving by doing this?

Reply:
I'm not saving anything. I don't really know why I did it like this in the first place. Will add relation to the team model